system_tests/resources/start-ec2-worker-vm-blueprint.yaml

```diff
@@ -60,12 +64,23 @@ node_templates:
       aws_config: *AWS_CONFIG
 
   server:
-    type: cloudify.aws.nodes.Instance
+    type: cloudify.aws.nodes.WindowsInstance
     properties:
       install_agent: false
       image_id: { get_input: image_id }
       instance_type: { get_input: instance_type }
       aws_config: *AWS_CONFIG
+      parameters:
+        user_data: |
+          <powershell>
+          &winrm quickconfig -q
```

Comment:
It's true that I need only one of those lines, but I never user userdata. I ran the line manually on the client, after it was up. Maybe when using userdata more commands are required.